src/Workspaces/CSharp/Portable/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -506,6 +506,17 @@ private static bool CanReplace(ISymbol symbol)
                         return true;
                     }
 
+                // If the parent is a conditional access expression, we could introduce an LValue
+                // for the given expression, unless it is a MemberBindingExpression.
+                // Case (1): In a parent expression of form "a?.Prop?.Length"
+                //                expression "a" can be replaced with an LValue
+                // Case (2): In a parent expression of form ".Prop?.Length" or ".Method()?.Length"
+                //                expression ".Prop" and ".Method()" cannot be replaced with an LValue.
+                case SyntaxKind.ConditionalAccessExpression:
+                    return !(expression.IsKind(SyntaxKind.MemberBindingExpression) || 
+                             expression.IsKind(SyntaxKind.InvocationExpression) && 
```

Comment:
Thanks, I'll look into that shortly. From the top of my mind, it will also need some additional checks, because the `Expression` part could start with a `MemberBindingExpression` if we're in a nested `ConditionalAccessExpression` and that can't be an `LValue`. 

I get your point though, I'll _try_ to express the conditional a bit more succinctly :smile: